src/main/java/dk/cngroup/lentils/entity/Image.java

```diff
@@ -16,15 +16,24 @@
     @Column(name = "image_id")
     private Long imageId;
 
-    @Column(name = "path")
-    @NotEmpty(message = "Musí být zadaná cesta k souboru")
-    private String path;
+    @Column(name = "imageUrl")
+    @NotEmpty(message = "Musí být zadaná cesta k obrázku")
+    private String imageUrl;
+
+    @Column(name = "isFromFile")
```

Comment:
Možná bych to pojmenoval jako is_remote/is_local nebo jen remote/local. A nechal to jako boolean,

Reply:
přejmenováno na local, změnila jsem i enum ImageSource (LOCAL, REMOTE, NONE)